Casks/chefdk.rb

```diff
@@ -2,7 +2,7 @@
   version "4.13.3"
   sha256 "a38e995e6a83856df64875dcc2424fe961b7dee9fca90a1290ddd1f01399ebf8"
 
-  url "https://packages.chef.io/files/stable/chefdk/#{version}/mac_os_x/#{MacOS.version}/chefdk-#{version}-1.dmg"
+  url "https://packages.chef.io/files/stable/chefdk/#{version}/mac_os_x/10.14/chefdk-#{version}-1.dmg"
```

Comment:
for MacOS 11 BigSur, we are supposed to use following binary:

CORRECT URL: https://packages.chef.io/files/stable/chefdk/4.13.3/mac_os_x/11.0/chefdk-4.13.3-1.dmg
INCORRECT URL : https://packages.chef.io/files/stable/chefdk/4.13.3/mac_os_x/11/chefdk-4.13.3-1.dmg

notice: 11.0, not 11

so form this fix, for MacOS 11, we should use 10.14 binary instead?